open-apparel-registry/open-apparel-registry
# Update openapparel.org references

## Overview

- Adds .opensupplyhub.org to the ALLOWED_HOSTS list in settings.py
- Changes DEFAULT_FROM_EMAIL in settings.py
- Adds opensupplyhub to CORS_ALLOWED_ORIGIN_REGEXES in settings.py
- Updates reports that exclude OAR emails with AND NOT u.email LIKE '%openapparel.org%' to also exclude opensupplyhub.org email addresses.
- Replaces openapparel.org with opensupplyhub.org in some tests and management commands, when seemed appropriate

Connects #2277 

## Testing Instructions

* Run `./scripts/test`
* Go to /admin/reports and view api_request_by_user, api_requests, average_matches_per_facility_by_type, contributor_first_appearance, contributors_with_active_facilities, countributors, facility_list_items_uploaded, facility_lists_uploaded, industry_data_vs_oar_data, industry_data_vs_oar_data_cumulative, mailing_list_recipients, non_public_contributors, percent_data_by_contributor_type, percent_data_by_contributor_type_cumulative, public_contributors, unique_contributor_count_by_type, and unique_public_list_count. Confirm they behave as expected. 

## Checklist
